scripts/handle-user-data

```diff
@@ -33,6 +35,10 @@ mount_data_partition() {
     mkdir -p /mnt/$PARTNAME/var/lib/boot2docker/etc/docker
     ln -s /mnt/$PARTNAME/var/lib/boot2docker/etc/docker /etc/docker
 
+    mkdir -p /mnt/$PARTNAME/var/lib/origin
+    mkdir -p /var/lib/origin
+    mount --bind /mnt/$PARTNAME/var/lib/origin/ /var/lib/origin/
```

Comment:
> If we go for this solution, we need to create a follow up issue to make sure to address this later.

@hferentschik right now I will vote for it. Issue which filled against origin also marked as P2 priority so hoping this will resolve soon.